src/codegen/revert.rs

```diff
@@ -38,15 +39,21 @@ pub enum SolidityError {
     String(Expression),
     /// The `Panic(uint256)` selector
     Panic(PanicCode),
+    /// User defined errors
+    Custom {
+        error_no: usize,
+        expr: Option<Expression>,
```

Comment:
I would expect this to be `Vec<Expression>` because a custom error can have any number of arguments.

Reply:
Ah right, this was a bit confusing, because the statement can be `revert("foo")` but also `revert Foo()`, I assumed in the case of a custom error it would capture the error (as a composite) instead individual arguments. Which is not the case, thanks for pointing this out